templates/3c_generate_qc_metrics.ipynb

```diff
@@ -95,10 +95,12 @@
    "id": "12a31336",
```

Comment:
<p>We plot the average over time to assess marker staining over the course of the run.</p><p>We are looking to make sure there there are not dramatic differences in staining intensity across the samples in the run, especially those that are correlated with run order, as this could indicate an instrument issue, rather than a biological differnece in signal</p><p>We expected some FOV-to-FOV variation in signal intensity, we're looking for large systematic differences that could idnicate a problem</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/toffy/pull/171/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>